src/index.html

```diff
@@ -1,132 +1,201 @@
 <!DOCTYPE html>
```

Comment:
I don't like the formatting, could you revert it?

Reply:
sure can, i forgot i had my code formatter on so i can revert it